### ISSUE TITLE
Cant SELECT over multiple regexps

### ISSUE THREAD

**Issue by mjdesa** [influxdb/influxdb#3129]
On OSX with Influx version 0.9 built from source

Running

```
select * from /^cpu/, /^mem/
```

yields

```
ERR: can not use field in group by clause: cpu
```

**Comment by beckettsean** [comment]
I think it's not possible to select from multiple measurements:

```
> show series
name: bar
---------
_key
bar


name: bat
---------
_key
bat

> show field keys 
name: bar
---------
fieldKey
value


name: bat
---------
fieldKey
fkey
```

`bar` and `bat` exist, no tags on either, one field on each.

```
> select * from bar, bat
ERR: unknown field or tag name in select clause: fkey
> select * from bat, bar
ERR: unknown field or tag name in select clause: fkey

> select value from bat, bar
ERR: unknown field or tag name in select clause: value
> select value from bar, bat
ERR: unknown field or tag name in select clause: value
```

**Comment by beckettsean** [comment]
@otoolep looks like another consequence of the implicit `GROUP BY *` with a `SELECT *`. I'm honestly not sure what the behavior is supposed to be when selecting from multiple measurements, but the regex examples seem to indicate that there's a `GROUP BY` happening under the hood for each field, whether or not that field exists on both measurements.

**Comment by otoolep** [comment]
I'll take a look during the query engine re-work.

**Comment by beckettsean** [comment]
`select` will work over multiple measurements if and only if they have identical field keys. Otherwise it will blow up with an error like the above. The regex in Michael's original example appears to expose both that issue and the implicit `group by *`

**Comment by beckettsean** [comment]
This is fixed in 0.9.3 nightly:

```
> show field keys 
name: thing
-----------
fieldKey
value


name: thingy
------------
fieldKey
bar

> select * from /thing/
name: thing
-----------
time			bar	value
1970-01-01T00:15:00Z		50
1970-01-01T00:15:10Z		60


name: thingy
------------
time				bar	value
2015-08-18T22:43:01.339541549Z	foo	

> 
```

It's hard to see in the output above, but both field sets are properly shown:

![screen shot 2015-08-18 at 15 52 49 1](https://cloud.githubusercontent.com/assets/6537329/9344936/35d8950a-45c1-11e5-939f-45e1d65ac7f2.png)
